lighthouse-core/gather/connections/connection.js

```diff
@@ -106,11 +106,15 @@ class Connection {
           {method: callback.method, params: object.result}, 'verbose');
         return object.result;
       }));
+    } else if (object.id) {
+      const error = object.error && object.error.message;
+      log.formatProtocol(`disowned method <= browser ${error ? 'ERR' : 'OK'}`,
+          {method: object.method, params: error || object.result}, 'verbose');
```

Comment:
> normalize object.method here rather than in formatProtocol?

I'd really rather not, that's exactly what got us into this mess of fatally erroring on unexpected input, unless I'm misunderstanding what you mean by "normalize" haha

> if it's an error, params probably shouldn't be set to the message?

where would you like to log the message then?

> and should it still throw if there's an error?

that would make these fatal in the future which seems bad. the point is that we don't care about these "disowned" (better name pending you 😄 ) events